Community.md

```diff
@@ -0,0 +1,47 @@
+
+Offhand, it's easy to imagine that this would make code quality suffer, but in reality it offers fresh perspectives to the codebase and encourages ownership from people who are depending on the project. If you are working with a codebase, then you probably have the skills to improve it, and offer valuable feedback.
+
+Everyone comes in with their own perspective on what a project could/should look like, and encouraging discussion can help expose good ideas sooner.
+
+#### Why do we give out push access?
+
+It can be overwhelming to suddenly be offered the chance to wipe the source code for a project. Don't worry, we don't let you push to master.
+
+All code is peer-reviewed, and we have the convention that someone other than the submitter should merge the pull request.
+
+As an organization contributor you can merge other people's pull requests, or other contributors can merge yours. You won't be assigned a pull request, but you're welcome to just jump in and take a code review on things that interest you.
+
+If it feels right, merge.
+
+This project is not continuously deployed, there is space for debate after review too. Offering the chance to revert, or make an amending pull request.
+
+It's convention to provide a call to action via `/cc @[org]/contributors` to invite discussion.
+
+#### How can we help you get comfortable contributing?
+
+It's normal for a first PR to be a potential fix for a problem, moving on from there to helping the project's direction can be difficult. We try to help contributors cross that barrier by offering good first step issues. These issues can be done without feeling like you're stepping on toes. Ideally, these are non-critical issues that are well defined. They will be purposely avoided by mature contributors to the project.
+
+We aim keep all project discussion inside GitHub issues. This is to make sure it is public. If you have questions about how to use the library, or how the project is running GitHub issues are the goto tool.
+
+#### Our expectations on you as a contributor
+
+To quote [@alloy](https://github.com/alloy) from [this issue](https://github.com/Moya/Moya/issues/135):
+
+> Don't ever feel bad for not contributing to open source.
+
+We want contributors to both provide ideas, keep the ship shipping and to take some of the load from others. It is non-obligatory; we're here to have fun. :trophy:
+
+If someone submits a pull request that's not perfect, and you are reviewing, it's better to think about the PR's motivation rather than the specific implementation. Having braces on the wrong line should not be a blocker. Though we do want to keep test coverage high, we will work with you to figure that out together.
+
+#### What about if you have problems that cannot be put into a public issue?
+
+Both [Ash Furrow](https://github.com/ashfurrow) and [Orta Therox](https://github.com/orta) have contactable emails on their GitHub profiles, and are happy to talk about any problems.
```

Comment:
No, I need to change this to be "here is an example version" - being that there needs to be a way that you can say you're available outside of public GH issues